workItems/workItemAccounts.go

```diff
@@ -27,13 +27,11 @@ func NewItemAccounts(
 // Save will save information about an account
 func (wiv *itemAccounts) Save() error {
 	accounts := make([]*data.Account, len(wiv.accounts))
```

Comment:
it is very important to change this line also: `accounts := make([]*data.Account, 0, len(wiv.accounts))` instead of `accounts := make([]*data.Account, len(wiv.accounts))` (mind the second parameter set to 0)

Reply:
fixed